pkg/api/azenvtypes.go

```diff
@@ -106,22 +106,14 @@ var (
 		ImageVersion:   "latest",
 	}
 
-	//SovereignCloudsUbuntuImageConfig is the Linux distribution for Azure Sovereign Clouds.
-	SovereignCloudsUbuntuImageConfig = AzureOSImageConfig{
+	//Ubuntu1804ImageConfig is the Ubunutu 18.04-LTS Linux distribution.
+	Ubuntu1804ImageConfig = AzureOSImageConfig{
 		ImageOffer:     "UbuntuServer",
-		ImageSku:       "16.04-LTS",
+		ImageSku:       "18.04-LTS",
 		ImagePublisher: "Canonical",
 		ImageVersion:   "latest",
 	}
 
-	//GermanCloudUbuntuImageConfig is the Linux distribution for Azure Sovereign Clouds.
-	GermanCloudUbuntuImageConfig = AzureOSImageConfig{
```

Comment:
changed the German cloud to use "latest" version since it was the only sovereign cloud that wasn't already doing that.